wallet-new/src/Cardano/Wallet/WalletLayer/Kernel/Transactions.hs

```diff
@@ -148,7 +148,7 @@ buildDynamicTxMeta assuranceLevel slotCount mSlotwithState currentSlot isPending
     let currentSlot' = flattenSlotId slotCount currentSlot
         goWithSlot confirmedIn =
             let confirmedIn'  = flattenSlotId slotCount confirmedIn
-                confirmations = currentSlot' - confirmedIn'
+                confirmations = (currentSlot' - confirmedIn') + 1
```

Comment:
I guess it would benefit from a comment explaining why is that (it's in the PR description, but that is not visible in the code 😉  )

Reply:
comment or a proper commit message!

Reply:
tbh this is just a fix for the integration tests. They check that when confirmed, the tx does not have confirmations=0. It would be more reliable if the checked the status of the tx (and not confirmations number). I will look into the old wallet to be sure the follow 1-indexed confirmations and I will add a comment.